pkg/hive/hive.go

```diff
@@ -372,7 +340,28 @@ func (s *Service) checkAndAddPeers(ctx context.Context, peers pb.Peers) {
 			mtx.Lock()
 			peersToAdd = append(peersToAdd, bzzAddress.Overlay)
 			mtx.Unlock()
-		}(p, cacheOverlay)
+		}()
+
+	}
+
+	for _, p := range peers.Peers {
+
+		multiUnderlay, err := ma.NewMultiaddrBytes(p.Underlay)
+		if err != nil {
+			s.metrics.PeerUnderlayErr.Inc()
+			s.logger.Error(err, "multi address underlay")
```

Comment:
This shouldn't be logged as an error if the node operator can't do anything with it, probably `Debug` is the best choice here.